app.ts

```diff
@@ -13,22 +13,23 @@ var compression = require('compression');
 var methodOverride = require('method-override');
 
 var config = require('./config/config.js');
-var auth = require('./backend/build/confluenceAuth.js');
 var api = require('./backend/build/routes');
 var adminApi = require('./backend/build/routes-admin');
 var checkerApi = require('./backend/build/routes-checker');
 
 var log = require('./backend/build/log.js');
 
+var auth = require('./backend/build/auth.js');
```

Comment:
I believe this could be without the ".js". Feel free to remove also from log-require.

Also, I wonder if requiring from `backend/build` is the best way for these, but that's another question...